Code_Saturne documentation for initial 6.0.5 installation

welucas2 opened the pull request:
This documentation is tested and provides the instructions needed to run Code_Saturne 6.0.5 on ARCHER2. Minor updates may be appropriate when I can install a GUI-enabled version.

kevinstratford reviewed (changes requested):
I'm not quite sure what the changes to `LD_LIBRARY_PATH` are doing in the script.
Should these be moved to the module file?

Do you need a `module restore /etc/cray-pe.d/PrgEnv-gnu` at the start instead?
Does this script really work?

welucas2 commented:
Thanks Kevin. Code_Saturne generates most of a script itself when intialising a case. I wrote this originally thinking users might want to just make the necessary changes to this script. I've rewritten slightly to make sure users know they can create a new one from scratch using the one provided. The script has also been updated:

- Fixed to work with the changes to modules yesterday night.
- Includes --export=none
- Adds a comment reminding the user to provide their budget code
- Doesn't have the LD_LIBRARY_PATH and cd lines. These were generated by Code_Saturne but redundant, so users can leave them if modifying, but a new script doesn't need them.
- Also removes CS_RET return code from the original Code_Saturne generated script. Possibly a bit unnecessarily confusing.

It's tested and works.
